### DISCUSSION TITLE
Creator Role "Consulting Writer"

### DISCUSSION THREAD

**Discussion by Zombanana** [bpepple/metron#240] [Q&A] [answered]
Hi,

First I want to say a big thanks to bpepple for creating metron and all other people who are involved in this project. :)
I am since a longer time not real happy with ComicVine metadata and looking for alternatives since a longer time.
Just wondering I just find now out about the project. The project looks very promising... seems like a dream become true :).

Now my first question:
I start my first contribution with the "Batman Eternal" Series:
https://metron.cloud/issue/batman-eternal-2014-1/

3x creators have according to issue #1 the role as "Consulting Writer".
![Screenshot 2024-04-30 181210](https://github.com/bpepple/metron/assets/97361360/fa49e518-e538-457d-bdb2-bde8b16b9550)
metron don't have this role for creators yet.

Is it possible to add this role? Should I use another creator role for the 3x consulting writers?
I have given them for now temporaly the role as "writer", until the question is closed.

I intend to continue the series, but waiting for anwer :).

Best Regards,

**Comment by bpepple** [comment]
I'm not opposed to adding it, but I do wonder how often it is used. It might make sense to use `Plot` as the role, what are your thoughts?

Oh, I went ahead and added the rest of the series since I wasn't aware you were going to continue working on it.

**Comment by Zombanana** [comment]
Hi bpepple,
I have seen allready that somebody was adding the rest. I just wanted to add the characters and everyathing was finished. That went so superfast?^^ I couldn't believe my eyes :). Thanks for finishing this up :).

I am not sure, if it is often used. But at this time, they had during the New 52 era alone 4 Batman series.
- Batman
- Batman & Robin
- Detective Comics
- The Dark Knight

So it made sense, to add the writer from all the series together as "Consulting Writers".
I am not sure if to add it. I am to unexperience with this and haven't seen this on any other series like this.

**Reply by bpepple** [reply, marked as answer]
One side effect of adding a new role, is that any comic tagger would need to update their role synonyms ([Comic-Tagger](https://github.com/comictagger/comictagger/blob/6ac2e326121bf582e38c61cf1b9179059989d897/comicapi/genericmetadata.py#L109), [Metron-Tagger](https://github.com/Metron-Project/darkseid/blob/b66e58088baec210f91426b35d0d691b7a03553e/darkseid/comicinfo.py#L40)) to link the new role to the `Writer` role otherwise no role would be added to the [ComicInfo.xml](https://anansi-project.github.io/docs/comicinfo/documentation#creator-fields).

Based on that, I'd lean towards not adding the `Consulting Writer` role. It would be easy to make this change to Metron-Tagger, but I'm not sure how quickly it would be added (or even implemented) on Comic-Tagger.

Now if some new metadata format were to created it might make sense to add the role, but so far there doesn't seem to be much community momentum on creating one.

**Comment by Zombanana** [comment]
Hi bpepple,

In this aspect you are right. Haven't thought about compability issues with other software. I think this should be a topic for this project:

https://anansi-project.github.io/docs/category/comicinfo

Comictagger, kavita, komga and Codex supporting the newer comicinfo Version as a new standard. I think it should maybe a topic for them to decision to keep a standard.

I don't know if it is possible to move this topic to this project, to bring it to their attention. Otherwise I think it can be closed on the metron hub for now?
